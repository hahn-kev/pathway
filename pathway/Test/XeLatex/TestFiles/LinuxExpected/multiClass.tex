\documentclass[a4paper]{article} 
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage{alphalph}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}
\usepackage{changepage}
\usepackage{needspace}

\parindent=0pt
\parskip=\medskipamount
\begin{document}
\pagestyle{plain}
\sloppy
\setlength{\parfillskip}{0pt plus 1fil}
\font\main="Charis SIL" at 12pt
\font\abcmain="Charis SIL":color=ffa500 at 24pt
\font\amain="Charis SIL":color=0000ff at 18pt
\font\bmain="Charis SIL":color=ff0000 at 12pt
\font\abenmain="Charis SIL":color=0000ff at 18pt
\font\acmain="Charis SIL":color=ffa500 at 18pt

\pagestyle{fancy} 

\abcmain{
sample text file b a c
}

\amain{
sample text file a
}

\bmain{
sample text file b
}

\abenmain{
sample text file a b
}

\amain{
sample text file b a
}

\acmain{
sample text file a c
}

\end{document}
