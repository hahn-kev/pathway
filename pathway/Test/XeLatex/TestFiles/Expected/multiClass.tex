\documentclass[a4paper]{article} 
\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage[margin=2cm,includeheadfoot]{geometry}

\begin{document} 
\pagestyle{plain} 
\font\acmain="Times New Roman":color=ffa500 at 18pt
\font\bmain="Times New Roman":color=ff0000 at 12pt
\font\amain="Times New Roman":color=0000ff at 18pt
\font\abcmain="Times New Roman":color=ffa500 at 24pt
\font\main="Times New Roman" at 12pt
\pagestyle{fancy} 

\abcmain{sample text file b a c }

\amain{sample text file a }

\bmain{sample text file b }

\amain{sample text file a b }

\amain{sample text file b a }

\acmain{sample text file a c }

\end{document}
